### ISSUE TITLE
Sample code errors

### ISSUE THREAD

**Issue by zomGreg** [areski/python-nvd3#92]
The instructions at:

[Introduction Page](http://python-nvd3.readthedocs.org/en/latest/introduction.html), specifically:

```
from nvd3 import pieChart

# Open File to write the D3 Graph
output_file = open('test-nvd3.html', 'w')

type = 'pieChart'
chart = pieChart(name=type, color_category='category20c', height=450, width=450)
chart.set_containerheader("\n\n<h2>" + type + "</h2>\n\n")

xdata = ["Orange", "Banana", "Pear", "Kiwi", "Apple", "Strawberry", "Pineapple"]
ydata = [3, 4, 0, 1, 5, 7, 3]

extra_serie = {"tooltip": {"y_start": "", "y_end": " cal"}}
chart.add_serie(y=ydata, x=xdata, extra=extra_serie)
chart.buildhtml()
output_file.write(chart.htmlcontent)

# close Html file
output_file.close()
```
render html that doesn't work due to some malformed cloudflare refs (missing "http:"):

```
<link href="//cdnjs.cloudflare.com/ajax/libs/nvd3/1.7.0/nv.d3.min.css" rel="stylesheet" />
<script src="//cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/nvd3/1.7.0/nv.d3.min.js"></script>
```
This happens if you do a:

```
pip install python-nvd3
```

The problem seems to be present only in the master branch, but seems fixed in develop, so if I PR a fix to the docs, the docs will be inaccurate when a new release goes. Thoughts?

**Comment by oz123** [comment]
We need to do a release. Coming soon...

**Comment by oz123** [comment]
released version 0.13.10. Should be fixed in there.

**Comment by zomGreg** [comment]
Confirmed. Looks good.
